settings.py

```diff
@@ -1076,6 +1076,17 @@ def _pkce_required(client_id):
 # used to override low-level index settings (number_of_replicas, number_of_shards, etc)
 ES_SETTINGS = None
 
+"""
+The CASE_SEARCH_SUB_INDICES should look like this:
+{
+    'co-carecoordination-perf': {
+        'index_cname': 'case_search_bha',
```

Comment:
where is the cname mapped to an actual index name?

Reply:
The [case_search_bha_adapter](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/es/case_search_bha.py#L36-L41) effectively maps cname to the actual index name. So using the cname defined in settings, we can grab the adapter as done here: https://github.com/dimagi/commcare-hq/pull/34673/files#diff-7ba2ac2cd9c42581b497ab0c7cb5ea77288d3a8d7e8583cfc2c35a4da9e063b6R232